tests/common/test_cgroupapi.py

```diff
@@ -62,8 +62,11 @@ def test_cgroups_should_be_supported_only_on_ubuntu_16_and_later(self):
             (['centos', '8.1', 'Source'], True),
             (['redhat', '8.2', 'Maipo'], True),
             (['redhat', '8.2.2111', 'Core'], True),
-            (['centos', '7.5', 'Source'], False),
-            (['redhat', '7.5', 'Maipo'], False),
+            (['centos', '7.4', 'Source'], True),
```

Comment:
We should change the name of this test; it indicates that only ubuntu versions are supported, (`test_cgroups_should_be_supported_only_on_ubuntu_16_and_later`).